EasyPost.Tests/OrderTest.cs

```diff
@@ -1,116 +1,121 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using EasyPost.Models.API;
+using EasyPost.Utilities.Annotations;
+using Xunit;
 
 namespace EasyPost.Tests
 {
-    [TestClass]
-    public class OrderTest
+    public class OrderTest : UnitTest
     {
-        private TestUtils.VCR _vcr;
-
-        [TestInitialize]
-        public void Initialize()
+        public OrderTest() : base("order")
         {
-            _vcr = new TestUtils.VCR("order");
         }
 
-        private static async Task<Order> CreateBasicOrder()
-        {
-            return await Order.Create(Fixture.BasicOrder);
-        }
+        #region CRUD Operations
 
-        [TestMethod]
+        [Fact]
+        [CrudOperations.Create]
         public async Task TestCreate()
         {
-            _vcr.SetUpTest("create");
-
-            Order order = await CreateBasicOrder();
-
-            Assert.IsInstanceOfType(order, typeof(Order));
-            Assert.IsTrue(order.id.StartsWith("order_"));
-            Assert.IsNotNull(order.rates);
-        }
-
-        [TestMethod]
-        public async Task TestRetrieve()
-        {
-            _vcr.SetUpTest("retrieve");
+            UseVCR("create");
 
             Order order = await CreateBasicOrder();
 
-            Order retrievedOrder = await Order.Retrieve(order.id);
-
-            Assert.IsInstanceOfType(retrievedOrder, typeof(Order));
-            // Must compare IDs since other elements of objects may be different
-            Assert.AreEqual(order.id, retrievedOrder.id);
+            Assert.IsType<Order>(order);
+            Assert.StartsWith("order_", order.id);
+            Assert.NotNull(order.rates);
         }
 
-        [TestMethod]
+        [Fact]
+        [CrudOperations.Read]
         public async Task TestGetRates()
         {
-            _vcr.SetUpTest("get_rates");
+            UseVCR("get_rates");
 
             Order order = await CreateBasicOrder();
 
-            await order.GetRates();
+            await order.GetRates(); // this does not return anything, does actually update in-place
```

Comment:
Question/Suggestion: Was the comment a personal reminder or meant to be kept in code? I'd suggest that the lack of a return value here signifies it returns nothing and therefore isn't a necessary comment.

Reply:
More of a personal reminder to myself, I guess, to remind myself about the unintuitive way we do in-place object updates, especially for a `getX` method.

Reply:
If it was a personal reminder, can you remove this?

Reply:
Resolved.